server/lib/data_import/my_anime_list.rb

```diff
@@ -31,15 +32,28 @@ def get_media(external_id) # anime/1234 or manga/1234
       end
     end
 
+    def get_character(external_id)
+      media = Mapping.lookup('myanimelist', external_id)
```

Comment:
I believe @NuckChorris means that you would now call it as `get_character(422)` as calling `get_character('character/422')` is a bit redundant.